app/src/main/java/com/tristanwiley/chatse/about/adapters/AboutVerticalIconAdapter.kt

```diff
@@ -8,17 +8,18 @@ import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import com.tristanwiley.chatse.R
+import com.tristanwiley.chatse.about.pokos.AboutIconPoko
 
 /**
- * Created by mauker on 31/08/17.
```

Comment:
Why remove my name Adam? :(

Reply:
Sorry I just replaced the default file template without thinking. :(